server/models/Team.js

```diff
@@ -195,10 +195,10 @@ Team.prototype.provisionFirstCollection = async function (userId) {
 };
 
 Team.prototype.addAdmin = async function (user: User) {
-  return user.update({ isAdmin: true });
+  return user.update({ isAdmin: true, isViewer: false });
 };
 
-Team.prototype.removeAdmin = async function (user: User) {
+Team.prototype.demoteUser = async function (user: User, to: string) {
```

Comment:
Argh, I hate these methods – why are they on team? Makes no sense – but not your fault 😉

Reply:
Do you feel better now?